tests/unit/main_html5/wrapper-event-tests.js

```diff
@@ -191,6 +191,31 @@ describe('main_html5 wrapper tests', function () {
     expect(vtc.notifyParameters).to.eql([vtc.interface.EVENTS.SEEKING]);
   });
 
+  it('should not raise seeking before initial time has seeked', function(){
+    vtc.interface.EVENTS.SEEKING = "seeking";
+    element.duration = 20;
+    spyOn(element.seekable, "start").andReturn(0);
+    spyOn(element.seekable, "end").andReturn(20);
+    element.seekable.length = 1;
+    wrapper.setInitialTime(10);
+    $(element).triggerHandler("seeking");
+    expect(vtc.notifyParameters[0]).to.not.eql(vtc.interface.EVENTS.SEEKING);
+    $(element).triggerHandler("seeked");
+    $(element).triggerHandler("seeking");
+    expect(vtc.notifyParameters[0]).to.eql(vtc.interface.EVENTS.SEEKING);
+  });
+
+  it('should raise seeking before initial time has seeked if initialtime is 0', function(){
```

Comment:
Previous one is for making sure the event is not raised before initial time; this one is for when initial time is 0 so we're already at this time before any seeked events are raised.  The difference is that for the first one we unblock seeking once seeked is raised.  in the second one seeking is never blocked